### PR TITLE
Adding LowPriorityCoreQuota and DedicatedCoreQuota

### PR DESCRIPTION
**Pull request by manishshar**
Adding LowPriorityCoreQuota and DedicatedCoreQuota in BatchAccountContext.cs

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Batch/Batch/Models/BatchAccountContext.cs [hunk 1]
```diff
@@ -110,6 +110,16 @@ public string TagsTable
         /// </summary>
         public int PoolQuota { get; private set; }
 
+        /// <summary>
```
Comment (markcowl) [inline src/Batch/Batch/Models/BatchAccountContext.cs]: We should modify a test to validate proper serialization/deserialization of the new parameters? Also, please add a changelog entry for this.  Note: if you are on an Azure team, please join the azure org (https://aka.ms/azuregithub)

Comment (markcowl) [reply src/Batch/Batch/Models/BatchAccountContext.cs]: Also, it looks like these properties do not exist, do you need to update the SDK?

**Comment by markcowl** [comment]
@manishar ping, are you actively working on this PR?

**Comment by manishshar** [comment]
Hi @markcowl , I checked sdk package 'Microsoft.Azure.Management.Batch' latest is 8.0.0.0. but In powershell latest version it is 4.2.1. So I reached to nuget team regarding the same and got reply from @matthchr that this is actually fixed in the latest preview package of powershell and they are waiting for the next breaking Powershell change to release it as non-preview though.

Is it possible if I can get GIThub repository link for "Microsoft.Azure.Management.Batch" version 4.2.1 under "azure-sdk-for-net"? Otherwise, I'm OK to close this PR as well.

**Pull request closed without merging**